src/java.base/share/classes/javax/crypto/CipherSpi.java

```diff
@@ -434,14 +437,14 @@ protected abstract void engineInit(int opmode, Key key,
      * them from {@code random}.
      *
      * <p>Note that when a {@code Cipher} object is initialized, it loses all
-     * previously-acquired state. In other words, initializing a cipher is
-     * equivalent to creating a new instance of that cipher and initializing
-     * it.
+     * previously-acquired state. In other words, initializing a {@code Cipher}
+     * object is equivalent to creating a new instance of that
+     * {@code Cipher} object and initializing it.
      *
-     * @param opmode the operation mode of this cipher (this is one of
+     * @param opmode the operation mode of this {@code Cipher} (this is one of
```

Comment:
add object here.

Reply:
Same comment appeared earlier. Odd. It's fixed.